.github/workflows/ci.yml

```diff
@@ -26,22 +26,19 @@ jobs:
         uses: conda-incubator/setup-miniconda@v2
         with:
           auto-update-conda: true
-          miniconda-version: latest
-          activate-environment: test
+          channels: conda-forge
+          mamba-version: '*'
+          activate-environment: s3fs-dev
           python-version: ${{ matrix.python-version }}
 
       - name: Install dependencies
         shell: bash -l {0}
         run: |
-          conda install -c conda-forge pip botocore aiobotocore "moto>=2.0" pytest flake8 black -y
-          pip install git+https://github.com/fsspec/filesystem_spec --no-deps
+          mamba env update -f requirements-dev.yml
```

Comment:
With this change, fsspec will default to the latest released version. I would prefer to install the current main branch in CI.

Reply:
should be fixed: https://github.com/fsspec/s3fs/runs/4239254873?check_suite_focus=true#step:4:558